.github/workflows/CI.yml

```diff
@@ -0,0 +1,127 @@
+name: CI
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
```

Comment:
these jobs will waste github actions' quota, even if it's free

I would prefer the CI to be just it is now

Reply:
Are you talking about the triggers?
This only allows you to run the workflow manually

Reply:
If you are talking about the jobs, the repeating parts use almost no quota compared to the rest.
And because we are testing on windows and linux it is quicker to just have one job that runs twice compared to running everything twice.